h/api/storage.py

```diff
@@ -33,15 +34,15 @@ def annotation_from_dict(data):
     return models.elastic.Annotation(data)
 
 
-def fetch_annotation(request, id, _postgres=None):
+def fetch_annotation(request, annotation_id, _postgres=None):
```

Comment:
[It's 'cos pylint doesn't like it](https://github.com/hypothesis/h/pull/3256/commits/831f6467f815c44f5abe56a50fefa5a2f67444db). Overriding a builtin does seem bad. On the other hand I've never actually used the `id` builtin, so I would not be against disabling this pylint warning